cybershell/orchestrator.py

```
@@ -0,0 +1,433 @@
+            if self.config.plugin.plugin_blacklist:
+                plugins = {k: v for k, v in plugins.items() 
+                          if k not in self.config.plugin.plugin_blacklist}
+            
+            return plugins
+        return {}
+    
+    def check_scope(self, target: str) -> bool:
+        """Check if target is in scope using unified config"""
+        return self.safety_config.is_in_scope(target)
+    
+    def fingerprint_target(self, target: str, use_cache: bool = True) -> TargetFingerprint:
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)